app/services/push_notifications/broadcast.rb

```diff
@@ -0,0 +1,29 @@
+module PushNotifications
+  class Broadcast
+    def initialize(push_notifications:)
+      @push_notifications = push_notifications
+    end
+
+    def send
+      return unless push_notifications.any?
+
+      client = Exponent::Push::Client.new
+
+      push_notifications.update_all(processed_at: Time.now.utc)
```

Comment:
I'd rather update the `processed_at` after publishing the notifications just in case that fails.

Reply:
We discussed about this and it's simpler this way, push notifications are not really important. You can loose one.